test/autoschedulers/anderson2021/test.cpp

```diff
@@ -14,7 +14,7 @@ int main(int argc, char **argv) {
     AutoschedulerParams params = {"Anderson2021", {{"parallelism", std::to_string(hardware_parallelism)}}};
 
     // Use a fixed target for the analysis to get consistent results from this test.
-    Target target("x86-64-linux-sse41-avx-avx2-cuda");
+    Target target("x86-64-linux-sse41-avx-avx2-cuda-debug");
```

Comment:
Unintentional? This isn't going to change anything in the current test, but if the test is changed to actually run code, it's going to invalidate any performance measurements.

Reply:
oops, yes, fixing